versions/1/data_stream/_dev/test/system/spec.yml

```diff
@@ -1,8 +1,35 @@
 spec:
   additionalContents: false
   contents:
-    - description: System test configuration
+    - description: Configuration of test case in yaml format.
       type: file
-      pattern: '^config.yml$'
+      pattern: '^test-[a-z0-9_.-]+-config.yml$'
       contentMediaType: "application/x-yaml"
-      required: false
\ No newline at end of file
+      required: false
+      content:
+        type: object
+        additionalProperties: false
+        properties:
+          input:
+            description: Input to use for test case.
+            type: string
+            additionalProperties: false
+          service:
```

Comment:
This makes sense at the moment since the only service deployer supported today is Docker Compose. But in the future we plan add other service deployers so we may need to revisit this property.